packages/ember-meta/lib/meta.ts

```diff
@@ -148,7 +148,7 @@ export class Meta {
     return this[key] || (this[key] = new Set());
   }
 
-  _getInherited(key: string) {
+  _findInherited1(key: string): any | undefined {
```

Comment:
since `_getInherited` and  `_finInherited` are almost same, just `_finInherited` accepts 2 keys and goes 2 keys deep into meta object, so renamed `_getInherited` to `_finInherited1` which goes one level deep and accepts 1 key, and renamed other appropriately 
`_finInherited1` - accepts 1 key goes one level deep
`_finInherited2` - accepts 2 key goes two level deep
`_finInherited3` - accepts 3 key goes three level deep

Reply:
this actually could be further refactored to have one `_finInherited` accepts list of keys that goes variable length deep, but I think that might cause performance issues but will check once this merged

Reply:
poking in case you still want to check for performance issues